### PR TITLE
Adding python API test part 5

### PR DESCRIPTION
**Pull request by Ying1123**
This PR(follow #6553) adds more functions and unit tests for python API.
Subsequent PR will include additional missing functions and unit tests.

1. Adding getNullSort() and mkEmptyBag() functions.
2. Allowing mkOp() with a list of arguments (previously allowed at most 2).
3. Allowing mkString() with additional boolean argument useEscSequences.
4. Corresponding changes to the tests.

### CODE REVIEW

File: src/api/python/cvc5.pxi [hunk 1]
```diff
@@ -686,28 +691,33 @@ cdef class Solver:
...
 
-        if arg0 is None:
+        if len(args) == 0:
             op.cop = self.csolver.mkOp(k.k)
-        elif arg1 is None:
-            if isinstance(arg0, kind):
-                op.cop = self.csolver.mkOp(k.k, (<kind?> arg0).k)
-            elif isinstance(arg0, str):
+        elif len(args) == 1:
+            if isinstance(args[0], kind):
```
Comment (yoni206) [inline src/api/python/cvc5.pxi:699]: I don't see a corresponding function in the cpp API. If the second argument exists, it is either a string, an int, or a vector, no?

File: src/api/python/cvc5.pxi [hunk 2]
```diff
@@ -686,28 +691,33 @@ cdef class Solver:
...
-                                 " mkOp: {}".format(" X ".join([str(k), str(arg0)])))
-        else:
-            if isinstance(arg0, int) and isinstance(arg1, int):
-                op.cop = self.csolver.mkOp(k.k, <int> arg0,
-                                                       <int> arg1)
+                                 " mkOp: {}".format(" X ".join([str(k), str(args[0])])))
+        elif len(args) == 2:
+            if isinstance(args[0], int) and isinstance(args[1], int):
+                op.cop = self.csolver.mkOp(k.k, <int> args[0],
+                                                       <int> args[1])
```
Comment (yoni206) [inline src/api/python/cvc5.pxi:717]: Formatting here looks a bit strange. Maybe fix manually or run `yapf` (at least on these lines).

**Review by yoni206 — CHANGES_REQUESTED** [review]
Thanks for doing this. Looks good, but I have couple of comments/questions inline.

File: src/api/python/cvc5.pxi (hunk 1 above)
Comment (Ying1123) [reply src/api/python/cvc5.pxi]: The old function have that case, I don't know why. Anyway I deleted it now.

File: src/api/python/cvc5.pxi (hunk 2 above)
Comment (Ying1123) [reply src/api/python/cvc5.pxi]: Addressed.

**Review by yoni206 — APPROVED** [review]

**Pull request merged**